docs/src/content/docs/guides/sidebar.mdx

```diff
@@ -0,0 +1,472 @@
+---
+title: Sidebar Navigation
+description: Learn how to set up and customize your Starlight site’s sidebar navigation links.
+---
```

Comment:
As it’s a new feature, how do you feel about us highlighting this page for a bit with a badge?

```suggestion
sidebar:
  badge: New
---
```